### ISSUE TITLE
There are problems with this document's schema impacting one or more items in the document

### ISSUE THREAD

**Issue by denious** [microsoft/vscode-azurearmtools#570]
Could be related to #536 and the recent 0.8.5 release.

![2020-04-02 19_58_55-cluster-template json - Deployment Scripts - Visual Studio Code  Administrator](https://user-images.githubusercontent.com/7120854/78310602-6c9d3e80-751c-11ea-861d-beb3a5159a1c.png)

There are no other errors in the document, this is the only one. Intellisense still works.
Error persists for all schema URL variations specified in the docs:
https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-syntax#template-format

Full Service Fabric ARM template where the issue is experienced:
https://1drv.ms/u/s!AtihpW2aJfOWsL8JxBxYORbgYpdi8g?e=00txJJ

**Comment by neilpeterson** [comment]
@denious thanks for the issue. The file location that you've provided is not resolving for me. Can you also verify that you are on the 0.8.5 version?

**Comment by denious** [comment]
Sorry for that; link updated!

Confirming 0.8.5:
![2020-04-02 20_39_51-cluster-template json - Deployment Scripts - Visual Studio Code  Administrator](https://user-images.githubusercontent.com/7120854/78312569-3a8edb00-7522-11ea-95c8-ce969a123718.png)

**Comment by neilpeterson** [comment]
Perfect thanks. It looks like something in the Microsoft.ServiceBus/namespaces/queues child resource is causing the issue. I will look into it.

**Comment by neilpeterson** [comment]
I think that I see what is going on here.

Several properties of the `Microsoft.ServiceBus/namespaces/queues` resource have a `format` of `duration`. It seems that the ARM Tools language server is not properly 'processing' or accounting for this format.

[Example](https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2017-04-01/Microsoft.ServiceBus.json#L1306)

I was able to validate this by commenting out all properties with this format. After doing so, the root schemas issue went away.

```
        {
            "type": "Microsoft.ServiceBus/namespaces/queues",
            "name": "test1234",
            "apiVersion": "2017-04-01",
            "properties": {
                "enableExpress": false,
                "enablePartitioning": false,
                // "autoDeleteOnIdle": "P30D"
                "maxDeliveryCount": 10,
                // "duplicateDetectionHistoryTimeWindow": "PT10M",
                "deadLetteringOnMessageExpiration": false,
                // "defaultMessageTimeToLive": "P1D",
                "requiresSession": false,
                "requiresDuplicateDetection": false,
                "maxSizeInMegabytes": 1024
                // "lockDuration": "PT5M"
```

I then identified other resources with a similar configuration, found a few, including `Microsoft.Resources/deploymentScripts`. Here is [an example](https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2019-10-01-preview/Microsoft.Resources.json#L165) from the deployment script schema.

This [template example for the deployment script resource](https://github.com/neilpeterson/arm-template-collection/blob/master/3-other/deployment-script/azuredeploy.json) reproduces the issue exactly: 

- issue on root schema 
- when I comment out any property with a property format of  `duration` the issue is removed

@alexgav to me this looks like an issue with how we are processing the schemas, and not the schemes themself. Can you confirm?

**Comment by StephenWeatherford** [comment]
@alexgav internal link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1092943

**Comment by alexgav** [comment]
Thanks Neil, good investigation. Unfortunately it is still a schema issue. Format "duration" was added in JSON schema draft 2019-09 (see https://json-schema.org/draft/2019-09/release-notes.html). Our tooling currently supports only Draft 04. So the schema needs to get updated to remove that format until our tooling is able to process newer schema. There is more info in the internal issue.

**Comment by StephenWeatherford** [comment]
Probably will be in the 0.9.1 release out soon

**Comment by StephenWeatherford** [comment]
Thanks for reporting!  The fix is in 0.9.1 which released this morning.
